docs/api-reference-1/wordpress-source.md

```diff
@@ -407,80 +450,108 @@ api.init({
 });
 ```
 
-#### `api.get({ endpoint, params, api?, isWpCom? })`
+#### `libraries.api.get()`
 
 Request entity from the WordPress REST API.
 
-**arguments**
+{% hint style="info" %}
 
-* `endpoint`: name of the endpoint if is a `/wp/v2` endpoint (e.g. `posts`), or the full path of other REST endpoints (e.g. `/frontity/v1/discovery`).
-* `params`: any parameter that will be included in the query params.
-* `api` (optional): overrides the value set with `api.set.`
-* `isWpCom` (optional): overrides the value set with `api.set.`
+`(options: object) => Promise`
 
-#### return
+- **Parameters**
+  - `options: object`
+    * **`endpoint`**: `string` Name of the endpoint if is a `/wp/v2` endpoint (e.g. `posts`), or the full path of other REST endpoints (e.g. `/frontity/v1/discovery`).
+    * **`params`**: `string` Any parameter that will be included in the query params.
+    * _`api`_: `string` _(optional)_ Overrides the value set with `api.set.`
+    * _`isWpCom`_: `boolean` _(optional)_ Overrides the value set with `api.set.`
+- **Return value**
+  - `Promise` Promise resolving to data requested
```

Comment:
I don't think the return value should be simply `Promise`. It needs to contain a reference to the value that the promise resolves to, which is the useful information for the developer.

In this case the promise resolves to a `Response` object from `fetch`: https://developer.mozilla.org/en-US/docs/Web/API/Response